### DISCUSSION TITLE
practice-exam/practice-exam-13

### DISCUSSION THREAD

**Discussion by giscus** [kananinirav/AWS-Certified-Cloud-Practitioner-Notes#286] [Comments]
# practice-exam/practice-exam-13

If you are preparing for AWS Cloud Practitioner Exam, this guide will help you with quick revision before the exam. I share the notes I used to study and pass exam.

https://kananinirav.com/practice-exam/practice-exam-13.html

**Comment by MahmoudHousam** [comment]
The correct answers for question 12 are B and E. Why not A? Amazon Route 53 is a scalable domain name system (DNS) web service. While it can be used to route traffic to both AWS and on-premises resources, it is not specifically designed for hybrid cloud architectures.
